src/main/java/org/swisspush/redisques/handler/RedisquesHttpRequestHandler.java

```diff
@@ -557,14 +628,19 @@ private void bulkDeleteQueues(RoutingContext ctx) {
                         return;
                     }
                     eventBus.send(redisquesAddress, buildBulkDeleteQueuesOperation(result.getOk()), (Handler<AsyncResult<Message<JsonObject>>>) reply -> {
-                        final JsonObject body = reply.result().body();
```

Comment:
Why remove this local variable?

Reply:
.result() can return null, so this will lead to a NPE.